libs/api/src/main/java/de/cyface/api/model/ClassifiedSegment.java

```diff
@@ -0,0 +1,434 @@
+     * @param version         The version of the format of this segment entry.
+     */
+    @SuppressWarnings("unused") // Part of the API
+    public ClassifiedSegment(final String oid, final boolean forward, final Geometry geometry, final double length,
+                             final Modality modality, final long vnk,
+                             final long nnk, final int start, final OffsetDateTime latestDataPoint, final String username,
+                             final double expectedValue, final double variance,
+                             final SurfaceQuality quality, final long dataPointCount, final String version) {
+        this.oid = oid;
+        this.forward = forward;
+        this.geometry = geometry;
+        this.length = length;
+        this.modality = modality;
+        this.vnk = vnk;
+        this.nnk = nnk;
+        this.start = start;
+        this.latestDataPoint = latestDataPoint;
+        this.username = username;
+        this.expectedValue = expectedValue;
+        this.variance = variance;
+        this.quality = quality;
+        this.dataPointCount = dataPointCount;
+        this.version = version;
+        Validate.isTrue(version.equals(CURRENT_VERSION));
+    }
+
+    /**
+     * Exports this measurement as GeoJSON feature.
+     *
+     * @param handler A handler that gets the GeoJson feature as string
+     */
+    public void asGeoJson(final Consumer<String> handler) {
+        // We decided to generate a String instead of using a JSON library to avoid dependencies in the model library
+
+        // measurement = geoJson "feature"
+        handler.accept("{");
+        handler.accept(jsonKeyValue("type", "Feature").getStringValue());
+        handler.accept(",");
+
+        // All tracks = geometry (MultiLineString)
+        handler.accept("\"geometry\":{");
+        handler.accept(jsonKeyValue("type", "MultiLineString").getStringValue());
+        handler.accept(",");
+        handler.accept("\"coordinates\":");
+        final var coordinates = convertToLineStringCoordinates(geometry);
+        handler.accept(coordinates);
+        handler.accept("},");
+
+        final var oid = jsonKeyValue("oid", getOid()); // for support
+        final var wayId = jsonKeyValue("@id", "way/232814001"); // FIXME: missing
+        final var highwayType = jsonKeyValue("highway", "residential"); // FIXME: missing
+        final var surfaceType = jsonKeyValue("surface", "asphalt"); // FIXME: missing
+        final var forward = jsonKeyValue("forward_moving", isForward());
+        final var modality = jsonKeyValue("modality", getModality().getDatabaseIdentifier());
+        final var maxLength = jsonKeyValue("max_length", getLength());
+        // FIXME: the start meter is sometime 0, 32, 82, ... and sometimes 0 50 100
+        // FIXME: the vnk/nnk is not the vnk/nnk of the WAY (is not consistent for subsequent segments)
+        // FIXME: the wayId is missing, maybe also add "highway" and "surface" from OSM
```

Comment:
I already created a bug-task for this stuff, I'll remove the comments together with the other ones, once I got a PR review.